### ISSUE TITLE
Accessibility problem: unlabeled buttons!

### ISSUE THREAD

**Issue by PrivacyDragon** [OxfordHCC/tracker-control-android#231]
Issue: The app list shows an app icon followed by it's name and sometimes some information thing. The app icon is a button. If you click on it, you disable internet access for that application. The problem is that the button is not labeled. A blind user can not hear what it is and what it does.

Fix: Add labels to the buttons. For example "Disable internet access for ..."

I might want to try to fix this myself if possible. If it is easy enough for me. It seems to me pretty easy, though sometimes I think something is easier than it actually is.

(I also posted this issue in my accessibility-hub repository: https://github.com/PrivacyDragon/FOSS-accessibility-hub/issues/7)

**Comment by kasnder** [comment]
I agree that this is really poor practice. Thanks for raising this!

**Comment by kasnder** [comment]
I've added it to the top of my todos.

**Comment by kasnder** [comment]
What needs to be done is adding a label to this image: https://github.com/OxfordHCC/tracker-control-android/blob/3abc89d53b4d99596b0f62ed04ec855f42b3edda/app/src/main/res/layout/rule.xml#L12

**Comment by kasnder** [comment]
Maybe, you could make a pull request, @PrivacyDragon ?

**Comment by PrivacyDragon** [comment]
I will try.
Thanks for pointing out where the image is located in the code.

**Comment by kasnder** [comment]
> I will try.
> Thanks for pointing out where the image is located in the code.

Great, thanks! Give it a try, otherwise I'm happy to look into it.

**Comment by kasnder** [comment]
@PrivacyDragon Thanks for the PR. I've merged this. I'll leave this open for further investigation.
